tests/data_context/store/test_store_backends.py

```diff
@@ -292,13 +292,16 @@ def test_TupleS3StoreBackend_with_empty_prefixes():
 
 
 def test_TupleGCSStoreBackend():
-    pytest.importorskip("google-cloud-storage")
+    # pytest.importorskip("google-cloud-storage")
     """
     What does this test test and why?
 
     Since no package like moto exists for GCP services, we mock the GCS client
     and assert that the store backend makes the right calls for set, get, and list.
+
+    TODO : One option may be to have a GCS Store in Docker, which can be use to "actually" run these tests.
```

Comment:
One option could be this repo [https://github.com/fsouza/fake-gcs-server](https://github.com/fsouza/fake-gcs-server)

Reply:
:+1: